src/math/mat3.js

```diff
@@ -1,243 +1,243 @@
-Object.assign(pc, (function () {

```

Comment:
Why has every line in this JS file been marked as changed? I'm a bit worried about merging if we make diffing harder.

Reply:
Because of line-endings. I've tried multiple times to commit only the few changed lines but for some reason the EOL is constantly being corrected. Is there something wrong with the EOLs in the original file?